Microsoft.DataBoxEdge RP: Correct API comments, correct extended info APIs

anponnet opened the pull request.

anponnet committed:
Microsoft.DataBoxEdge RP: Correct API comments, correct extended information APIs

shahabhijeet reviewed (dismissed):
Update nuget version in csproj
Also update version in AssemblyInfo
Also generate sdk using generate.ps1, you are not generating the metadata that is required.

anponnet committed:
Updated the nuget and library versions

anponnet commented:
> Update nuget version in csproj
> Also update version in AssemblyInfo
> Also generate sdk using generate.ps1, you are not generating the metadata that is required.

1. Updated nuget version
2. Updated version in AssemblyInfo
3. Generated using generate.ps1 only - this is in the file "Start-AutoRestCodeGeneration -ResourceProvider "edgegateway/resource-manager" -AutoRestVersion "latest"
"

anponnet commented:
> Update nuget version in csproj
> Also update version in AssemblyInfo
> Also generate sdk using generate.ps1, you are not generating the metadata that is required.

@shahabhijeet, Can you please check this?

anponnet commented:
@shahabhijeet , Can you please merge this PR?

dsgouda commented on src/SDKs/EdgeGateway/Management.EdgeGateway/Microsoft.Azure.Management.EdgeGateway.csproj:
```diff
@@ -7,7 +7,7 @@
-    <Version>0.8.0-preview</Version>
+    <Version>0.8.1-preview</Version>
     <PackageTags>Microsoft Azure EdgeGateway management;EdgeGateway;EdgeGateway management;</PackageTags>
     <PackageReleaseNotes>
```
Please update `PackageReleaseNotes`

dsgouda reviewed (changes requested):
Looks great apart from a minor comment.

anponnet committed:
Updated package release notes

dsgouda reviewed (approved):
LGTM

dsgouda commented:
Will merge once CIs pass
